pkgs/top-level/all-packages.nix

```diff
@@ -5384,6 +5384,8 @@ with pkgs;
     protobuf = protobuf_21;
   };
 
+  ghidra-extensions = recurseIntoAttrs (callPackage ../tools/security/ghidra/extensions.nix { });
```

Comment:
@lf- since we are callPackaging `ghidra-extensions` in `with-extensions.nix` anyway, should this still be exposed here?

Reply:
this ensures that the extensions will be indexed in the package list and eg. on search.nixos.org